unicodetools/org/unicode/tools/emoji/candidateData.txt

```diff
@@ -120,34 +120,63 @@ Proposal=L2/19-396, L2/19-400
 After=🤔
 U+102000
 Name=saluting face
-Keywords=okay | pride | salute | sunny | troops | yes
+Keywords=okay | pride | respect | salute | troops | yes
 
 Proposal=L2/19-378
 After=🤔
 U+102001
 Name=face with peeking eye
 Keywords=captivated | I can’t look away | peeking | peeping | staring
 
-Proposal=L2/19-380
-After=💔
-U+102002
-Name=mending heart
-Keywords=healthy | improving | mend | recovering | recuperating | well
-
 Proposal=L2/19-371
 After=💮
-U+102003
+U+102002
 Name=lotus
 Keywords=bloom | Buddhism | flower | Hinduism | lotus | purity
 
 Proposal=L2/19-375
 After=🌺
-U+102004
+U+102003
 Name=orchid
 Keywords=beauty | bloom | flower | orchid | rare
 
 Proposal=L2/19-379
 After=🩹
-U+102005
+U+102004
 Name=crutch
 Keywords=cane | crutch | disability | mobility aid | walking | walking stick
+
+#================================
+Status=Draft Candidate
+#================================
+
+Proposal=L2/20-066
+After=😬
+U+1F62E U+200D U+1F4A8
+Name=face exhaling
+Keywords=exhale | gasp | groan | relief | whisper | whistle
+
+Proposal=L2/20-131
+After=😵
+U+1F635 U+200D U+1F4AB
+Name=face with spiral eyes
+Keywords=dizzy | hypnotized | spiral | trouble | whoa
+
+Proposal=L2/19-380
+After=💔
+U+2764 U+FE0F U+200D U+1FA79
+Name=mending heart
+Keywords=healthier | improving | mending | recovering | recuperating | well
+
+Proposal=L2/20-128
+After=💔
```

Comment:
Shouldn't be an issue that it's the same `After=` as above.

Reply:
Ken Lunde confirmed [here](https://docs.google.com/document/d/1sfPeetALAwcytdLx6vc-HG5N2vsHhlhuklww8d_2S2g/edit?disco=AAAAGgWcgxI).